kernel/private/classes/clusterfilehandlers/dfsbackends/mysqli.php

```diff
@@ -1638,6 +1638,16 @@ public function _startCacheGeneration( $filePath, $generatingFilePath )
         if ( !$this->_query( $query, "_startCacheGeneration( $filePath )", false ) )
         {
             $errno = mysqli_errno( $this->db );
+
+            //once deadlock found / lock wait timeout found, we return ko status which will force read from stale cache
```

Comment:
super nitpick, but as seen on line further down we always have a whitespace before the comment text